rmp-serde/src/decode.rs

```diff
@@ -184,6 +189,38 @@ impl<R: Read> Deserializer<ReadReader<R>> {
     }
 }
 
+impl<R: Read, C: SerializerConfig> Deserializer<R, C> {
+    /// Consumes this deserializer and returns a new one, which will deserialize types with
+    /// human-readable representations (`Deserializer::is_human_readable` will return `true`).
+    ///
+    /// This is primarily useful if you need to interoperate with serializations produced by older
+    /// versions of `rmp-serde`.
+    pub fn with_human_readable(self) -> Deserializer<R, HumanReadableConfig<C>> {
+        let Deserializer { rd, config, marker, depth } = self;
+        Deserializer {
+            rd,
+            config: HumanReadableConfig::new(config),
+            marker,
+            depth,
+        }
+    }
+
+    /// Consumes this deserializer and returns a new one, which will deserialize types with
+    /// binary representations (`Deserializer::is_human_readable` will return `false`).
+    ///
+    /// This is the default MessagePack deserialization mechanism, consuming the most compact
+    /// representation.
+    pub fn with_binary(self) -> Deserializer<R, BinaryConfig<C>> {
```

Comment:
shouldn't this be a breaking change due to the default being change from `human_readable: true` to now `human_readable: false`?

Reply:
I can bump the version to 0.15.0 if you prefer.